### ISSUE TITLE
Please undeprecate! 😩

### ISSUE THREAD

**Issue by fyodorio** [etkachev/nx-webstorm#116]
**Is your feature request related to a problem? Please describe.**
The official nx's plugin is a total unusable garbage, and it kills the IDE periodically 🤦‍♂️

**Describe the solution you'd like**
Bring back this plugin 😄

**Describe alternatives you've considered**
I use CLI now inevitably and need to dig out my workspace genrators each time to come up with a proper call

**Additional context**
Man I miss the simplicity of working with this plugin!

**Comment by etkachev** [comment]
oh no! Sorry to hear that. What specific issues are you running into? I've been using the official plugin and so far most issues have been resolved, and working smoothly. Let me know. Would love to help out, also not sure when I would have time to dig back in to get this working with latest webstorm versions 😅

**Comment by fyodorio** [comment]
@etkachev yeah, I got it, that was just a kind of appreciation rant. The issues... Well, I'm not going to bother you, I tried with nx guys but they're too busy I guess. Anyway, I'm on a project where nx wasn't necessary initially but was added nevertheless and now the team pays a pretty dime for that 😅

Not talking about the functional issues though, the plugin itself seems to have a bunch of _compatibility_ issues in WebStorm, that's a big deal, as it causes crashes even when you don't touch it.

I came to using good ol' command line and a set of predefined and well-documented curses. Eventually, GUI is a bad pattern for developer, we need to get away from it...

**Comment by etkachev** [comment]
Yea I tried to keep up with all the new structure of how custom generators work with the new nx versions, but it became difficult overtime. Thus leaving it as is. Our project in the past did have issues because of invalid format on the schemas for our project.json or angular.json files. Issue related here https://github.com/nrwl/nx-console/issues/1768

We had too many libraries and apps to go through manually, but here is a helpful webstorm tool you can try to see if that is the case with yours.

1. so I created a new intellij scope that targets project.json files like this
<img width="1187" alt="image" src="https://github.com/etkachev/nx-webstorm/assets/9970746/37b7284e-172a-4622-96ff-fca92829982f">
2. then I search new action for Run Inspections my name... and selected the one for json schema compliance
<img width="1192" alt="image" src="https://github.com/etkachev/nx-webstorm/assets/9970746/f62fa250-4bd2-4578-b9a3-9dfb9b53ba4b">
3. then I selected the new custom scope I created earlier for all project.json files
<img width="1193" alt="image" src="https://github.com/etkachev/nx-webstorm/assets/9970746/0b0b0591-9ca6-40f1-af51-05546787196a">
4. you could run it through the whole project, but this isolates where the problem could be, and because our workspace is so huge

For us the issue was one of the project.jsons seemed to be a bad merge of implicitDependencies that slipped into the targets. Once that was fixed, then the plugin started working as expected.
